src/System.Collections/tests/Generic/SortedSet/SortedSet.TreeSubSet.Tests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Collections.Tests
+{
+    public class SortedSet_TreeSubset_Int_Tests : SortedSet_TreeSubset_Tests<int>
+    {
+        protected override int Min => int.MinValue;
+        protected override int Max => int.MaxValue;
+        private int Current { get; set; } = 1;
+
+        protected override bool DefaultValueAllowed => true;
+
+        protected override int CreateT(int seed)
+        {
+            return Current++;
```

Comment:
I have based it off the seed for the `int` tests.
However, for the `string` tests, I get several hundred exceptions thrown like this during the tests:
>  System.ArgumentOutOfRangeException : Specified argument was out of the range of valid values.